.github/workflows/iroha2-release-pr.yml

```diff
@@ -133,9 +133,15 @@ jobs:
       image: hyperledger/iroha2-ci:nightly-2023-06-25
     steps:
       - name: Maximize build space
-        run: |
-          sudo rm -rf /usr/share/dotnet
```

Comment:
there is 3 other places in this file and one in `workflows/iroha2-dev-pr-ui.yml` that you should also apply this to

Reply:
We _might_ do it, but actually it's not mandatory because only `long` tests job is in a lack of space. From the other hand, this disk free up Action takes a several minutes to run.

Reply:
> From the other hand, this disk free up Action takes a several minutes to run.

and if you enabled only `dotnet: true` and `tool-cache`? would it still take several minutes to run?

Reply:
It takes 0-1 seconds according to workflow timestamp. Specific action takes up to 3 minutes judging by the documentation.